### PR TITLE
Pixel: update font sizes and weights

### LINKED ISSUE #6454: Pixel: Styles: Typography — Sizes & Weights
Update Typography settings:

Scope:
* Font Weight
* Font Size (Fluid typography)

Blocks:
* Paragraphs
* Headdings
* Site-title
* Post-title

Designs: Cm51iCQSPu8Ar1qufDTeOQ-fi

image

### PR DESCRIPTION
**Pull request by jffng**
#### Changes proposed in this Pull Request:

This PR updates font sizes and weights for Pixel.

#### Related issue(s):

Closes #6454

### CODE REVIEW

File: pixel/theme.json [hunk 1]
```diff
@@ -369,32 +371,34 @@
...
             "h4": {
                 "typography": {
-                    "fontSize": "var(--wp--preset--font-size--medium)"
+                    "fontSize": "1.125rem"
                 }
             },
             "h5": {
                 "typography": {
-                    "fontSize": "1.125rem"
+                    "fontSize": "0.813rem",
```
Comment (vcanales) [inline pixel/theme.json:407]: What is the source for sizes and clamp ranges?

In the Figma file, for instance, `h5` is set to `14px`, so in theory this should be `1rem` if we don't change the current root font size.

Comment (jffng) [reply pixel/theme.json:407]: The root font size is `16px`, this should be standard and unchanged unless I'm missing something. 

Sorry, should have specified — the clamp range is based on a 320px - 1280px range.

**Review by vcanales — APPROVED** [review]
Looks good!

File: pixel/theme.json (hunk 1 above)
Comment (madhusudhand) [reply pixel/theme.json:407]: [Gutenberg](https://github.com/WordPress/gutenberg/blob/trunk/lib/block-supports/typography.php#L372) is always assuming min as `48rem` and max as `100rem` for the fluid typography. I guess it is best to use the same range.

File: pixel/theme.json [hunk 2]
```diff
@@ -150,17 +150,17 @@
...
                 {
                     "name": "Small",
-                    "size": "1rem",
+                    "size": ".813rem",
                     "slug": "small"
                 },
                 {
                     "name": "Medium",
-                    "size": "1.5rem",
+                    "size": "clamp(1rem, 0.8611111111111112rem + 0.6944444444444444vw, 1.125rem)",
```
Comment (madhusudhand) [inline pixel/theme.json]: lets enable fluid typography for the standard sizes?

File: pixel/theme.json [hunk 3]
```diff
@@ -150,17 +150,17 @@
...
                     "slug": "medium"
                 },
                 {
                     "name": "Large",
-                    "size": "1.75rem",
+                    "size": "1.5rem",
```
Comment (madhusudhand) [inline pixel/theme.json:170]: when fluid is enabled globally, and explicit `fluid: false` is required here to apply fixed size.

File: pixel/theme.json [hunk 4]
```diff
@@ -369,32 +371,34 @@
...
-                    "fontSize": "1.125rem"
+                    "fontSize": "0.813rem",
+                    "textTransform": "uppercase"
                 }
             },
             "h6": {
                 "typography": {
-                    "fontSize": "var(--wp--preset--font-size--small)"
+                    "fontSize": "0.75rem",
+                    "textTransform": "uppercase"
```
Comment (madhusudhand) [inline pixel/theme.json:415]: update the font weight of `heading` to `500` and override it to `400` in `h6`

File: pixel/theme.json [hunk 5]
```diff
@@ -333,8 +334,9 @@
             },
             "core/site-title": {
                 "typography": {
-                    "fontSize": "1.125rem",
+                    "fontSize": "0.875rem",
                     "fontWeight": "700",
```
Comment (madhusudhand) [inline pixel/theme.json]: update weight to 400.

File: pixel/theme.json (hunk 2 above)
Comment (jffng) [reply pixel/theme.json]: Done in https://github.com/Automattic/themes/pull/6479/commits/60b9d74f46b65b3dd39265c9acdabcc2d136e1ba.

File: pixel/theme.json (hunk 4 above)
Comment (jffng) [reply pixel/theme.json:415]: Done in https://github.com/Automattic/themes/pull/6479/commits/b4d39e520493859d023dff9835254f388d6fa2b6.

File: pixel/theme.json (hunk 5 above)
Comment (jffng) [reply pixel/theme.json]: Done in https://github.com/Automattic/themes/pull/6479/commits/8755037c0fcabba64214d4efa6096182cc452597.

**Comment by jffng** [comment]
@madhusudhand I made the updates suggested, including updating the heading font sizes to use the same viewport bounds as Gutenberg's fluid type scale. Could you give this another review?

**Review by madhusudhand — APPROVED** [review]
LGTM!

**Pull request merged**